requirements.txt

```diff
@@ -17,7 +17,7 @@ geopy==1.11.0
 gunicorn==19.8.0
 humanize==0.5.1
 idna==2.6
-kombu==4.1.0
+kombu<4.2
```

Comment:
Can we just remove this line altogether? Superset doesn't directly use `kombo` it and it was added in part to circumvent the issue. Also packages in `requirements.txt` should be pinned to a specific version.

cc: @timifasubaa